app/helpers/otp_helper.rb

```diff
@@ -0,0 +1,44 @@
+module OtpHelper
+  def phone_confirmation_instructions
+    if @delivery_method == 'sms'
+      t('instructions.2fa.confirm_code_sms', number: @phone_number)
+    elsif @delivery_method == 'voice'
+      t('instructions.2fa.confirm_code_voice', number: @phone_number)
+    end
+  end
+
+  def alternative_2fa_links
+    case @delivery_method
+    when 'sms'
+      "#{voice_fallback_link}#{totp_option_link}."
```

Comment:
Is this a concern even if we control the strings? As in, there is no user supplied data? Just curious!